### ISSUE TITLE
Allow program parameters to modify globals

### ISSUE THREAD

**Issue by tbarchyn** [IM3S/LDAR_Sim#92]
With the previous 'P_*.txt' style of parameter files the first program would set the number of timesteps and a bunch of other global parameters.

This functionality is slated for depreciation with PR #90 but I think it is something people want to use.

If so, we likely need a sketch of how this is intended to work and the use case for doing this.

Which program would overwrite the globals? All of them? A certain designated one?
What kind of warnings should be emitted?
How to document this so onboards know how to enable / disable this functionality?

@tarcadius @KeeganShaw-GIS

**Comment by tarcadius** [comment]
Yeah, this is definitely important, but complex to communicate and could also create problems unknowingly. Yes - warnings would be crucial. 

Could we just use the global input file to do this? Or does it have to be program level? This is coming back to the problem that "global parameters" and "program parameters" are potentially really just the same thing....
